.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+#### Proposed Changes
+
+*
+
+#### Testing Instructions
+
+*
+
+#### Fixes \#
```

Comment:
Should've this been just `#`, not `\#`?

Reply:
the backslash escapes the heading hashes in front else the last hash won't show. I now see that can be confusing. I'll update it to just `Fixes #`